### PR TITLE
remove deprecated ioutil usages

### PR DESCRIPTION
**Pull request by ramaraochavali**
ioutil package has been deprecated in 1.16 and functions have been moved to io and os packages. This PR changes those usages

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: tests/integration/pilot/multi_version_revision_test.go [hunk 1]
```diff
@@ -165,6 +169,33 @@ func installRevisionOrFail(t framework.TestContext, version string, configs map[
 	}
 }
 
+// ReadInstallFile reads a tar compress installation file from the embedded
+func ReadInstallFile(f string) (string, error) {
```
Comment (howardjohn) [inline tests/integration/pilot/multi_version_revision_test.go:173]: I think this is a merge conflict

**Review by howardjohn — APPROVED** [review]

File: tests/integration/pilot/multi_version_revision_test.go (hunk 1 above)
Comment (ramaraochavali) [reply tests/integration/pilot/multi_version_revision_test.go]: great catch. I do not know how you caught this in these 200 file changes

**Review by shamsher31 — APPROVED** [review]

**Comment by ramaraochavali** [comment]
@istio/wg-policies-and-telemetry-maintainers @istio/wg-docs-maintainers can you PTAL?

**Review by morvencao — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Comment by ramaraochavali** [comment]
@istio/wg-policies-and-telemetry-maintainers can you PTAL?

**Review by bianpengyuan — APPROVED** [review]

**Pull request merged**